package-lock.json

```diff
@@ -3666,7 +3666,7 @@
         },
         "readable-stream": {
           "version": "1.0.34",
-          "resolved": "https://registry.npmjs.org/readable-stream/-/readable-stream-1.0.34.tgz",
+          "resolved": "http://registry.npmjs.org/readable-stream/-/readable-stream-1.0.34.tgz",
```

Comment:
This is a vestige of an old npm install from a version that's since been fixed. Can you try deleting your `node_modules` and rerunning `npm install`? IIRC that should avoid regressing these to `http`.

Reply:
There apparently gremlins in `npm`. It will always try `https` first. To actually fix, I had to force clean, delete the `package-lock.json` and reinstall.

Reply:
but it doesn't look fixed? these are still marked as http. On latest npm?

Reply:
🙁 yes. Was on node 12 and did `npm i -g npm` then regenerated the lock file.